### PR TITLE
[iOS] Adds getIsRecording method

### PR DESCRIPTION
**Pull request by jasonmerino**
Needed to get current recording status of Lookback.io SDK for the case where the user does not accept the terms and conditions and the recording doesn't actually start.

### CODE REVIEW

File: ios/RNLookback.m [hunk 1]
```diff
@@ -82,4 +82,9 @@ - (dispatch_queue_t)methodQueue {
   [[Lookback sharedLookback] exitedView:view];
 }
 
+// check if lookback is recording
+RCT_REMAP_METHOD(getIsRecording, resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    resolve([NSNumber numberWithBool:[[Lookback sharedLookback] isRecording]]);
```
Comment (negativetwelve) [inline ios/RNLookback.m]: can you match the rest of the file's indentation? we use two spaces everywhere, thanks!

File: readme.md [hunk 2]
```diff
@@ -79,7 +79,12 @@ Lookback.showLookbackUploads();
 Lookback.enteredView('my-view');
 
 // Track exiting a custom view.
-Lookbac.exitedView('my-view');
```
Comment (negativetwelve) [inline readme.md:82]: good catch!

**Review by negativetwelve — CHANGES_REQUESTED** [review]
thanks for the PR @jasonmerino! Just a small formatting issue then we can merge this in. Does this require an SDK version bump?

**Comment by jasonmerino** [comment]
Okay, I believe I've fixed up the formatting to match.

I don't think there needs to be any bump of the Lookback SDK. Our podfile says we've got `"LookbackSafe", "~> 1.4.1"` installed. I'm not sure if the `isRecording` getter is a new method to the Lookback SDK though.

As far as NPM goes, it's additive and I didn't change any of the existing methods, so it should be good to go out as a minor release there.

**Comment by negativetwelve** [comment]
looks good! Release on npm as `1.3.1`

**Pull request merged**